developer/pages-router/installation.mdx

```diff
@@ -2,14 +2,15 @@
 title: Installation
 ---
 
-import RuntimeExample from "/snippets/reference/runtime.mdx"
-import ClientExample from "/snippets/reference/client.mdx"
-import ComponentsExample from "/snippets/reference/components.mdx"
-import NextConfigExample from "/snippets/reference/next-config.mdx"
-import PagesCatchAllExample from "/snippets/pages-router/catch-all.mdx"
-import DocumentExample from "/snippets/pages-router/document.mdx"
-import ApiHandlerExample from "/snippets/pages-router/api-handler.mdx"
-import PagesAppExample from "/snippets/pages-router/pages-app.mdx"
+import RuntimeExample from "/snippets/reference/runtime.mdx";
+import ClientExample from "/snippets/reference/client.mdx";
+import ComponentsExample from "/snippets/reference/components.mdx";
+import NextConfigExample from "/snippets/reference/next-config.mdx";
+import PagesCatchAllExample from "/snippets/pages-router/catch-all.mdx";
+import DocumentExample from "/snippets/pages-router/document.mdx";
+import ApiHandlerExample from "/snippets/pages-router/api-handler.mdx";
+import PagesAppExample from "/snippets/pages-router/pages-app.mdx";
```

Comment:
All these changes also seem like they wouldn't happen if we configured Prettier.

Reply:
We do actually have prettier configured. We just need to have it format all docs in a separate pr

Reply:
Ah, got it. We should probably also install a local version of Prettier so that it's not using the one that's shipped with the VS Code extension

Reply:
Cool, will do! Thanks!